### PR TITLE
docs: add contributing guide

### PR DESCRIPTION
**Pull request by aladdin-add**
##### Checklist

- [x] `npm test` passes
- [ ] tests are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

##### Description of change

- any feature?
- close https://github.com/umijs/qiankun/ISSUE_URL

### CODE REVIEW

**Review by kuitos — COMMENTED** [review]
Nice~ 英文版本也来一个？

**Comment by aladdin-add** [comment]
先看一下中文的，没问题的话再让chatgpt 翻译一个 :)

**Comment by kuitos** [comment]
commit msg 规范补充一个？用 Conventional Commits 规范就行

**Comment by kuitos** [comment]
后面统一成Conventional Commits就好了，emoji是我个人喜好

File: README.md [hunk 1]
```diff
@@ -107,6 +107,10 @@ Visit `http://localhost:7099`.
 
 See [Qiankun 3.0 Roadmap](https://github.com/umijs/qiankun/discussions/1378)
 
+## 🤝 Contributing
+
+See [contributing guide](./CONTRIBUTING-en.md).
```
Comment (kuitos) [inline README.md:112]: 默认英文的话文件名可以改成  CONTRIBUTING.md

Comment (aladdin-add) [reply README.md]: done~

**Review by kuitos — APPROVED** [review]
Nice work~

**Pull request merged**